src/index.html

```diff
@@ -1,13 +1,464 @@
+              <li class="navbar-mobile__item"><a href="#technology" class="navbar-mobile__link">TECHNOLOGY</a></li>
+              <li class="navbar-mobile__item"><a href="#features" class="navbar-mobile__link">FEATURES</a></li>
+              <li class="navbar-mobile__item"><a href="#contact" class="navbar-mobile__link">GET IN TOUCH</a></li>
+            </ul>
+            <div class="nav-top-actions">
+              <a href="#specs" class="nav-top-actions__specs">SPECS</a>
+              <span>
+                <a class="nav-top-actions__lang nav-top-actions__lang--active" href="#en">EN /</a>
+                <a class="nav-top-actions__lang" href="#ua"> UA</a>
+              </span>
+              <a class="button button--primary button--color--black" href="http://crazybaby.com/">BUY</a>
+            </div>
+          </nav>
+        </div>
+      </div>
+      <div class="header__content">
+        <div class="header__info">
+          <h1 class="header__title">
+            Futuristic
+            <br>Wireless
+            <br>Speaker
+          </h1>
+          <p class="header__paragraph">
+            Luna’s performance is balanced and smooth in all frequency ranges which makes the music both naturally pleasant and distinctly more layered.
+          </p>
+
+        </div>
+
+        <picture class="header__picture">
+          <source media="(min-width: 1025px)" srcset="./images/desktop/1.png">
+          <source media="(min-width: 641px)" srcset="./images/tablet/1.png">
+          <img src="./images/mobile/1.png" alt="1">
+        </picture>
+      </div>
+
+      <!-- DESKTOP NAV-BAR:  -->
+      <nav class="page__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#story" class="nav__link">OUR STORY</a></li>
+          <li class="nav__item"><a href="#about-us" class="nav__link">ABOUT US</a></li>
+          <li class="nav__item"><a href="#technology" class="nav__link">TECHNOLOGY</a></li>
+          <li class="nav__item"><a href="#features" class="nav__link">FEATURES</a></li>
+          <li class="nav__item"><a href="#contact" class="nav__link">GET IN TOUCH</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page__main">
+      <section id="story" class="page__section page__section--container">
+        <!-- BENEFITS MOBILE-SLIDER:-->
+        <div class="benefits-mobile">
+          <div class="benefits-mobile__content">
+            <div class="benefits-mobile__item benefits">
+              <img
+                class="benefits__img"
+                src="./images/compass.svg"
+                alt="compass"
+              >
+              <h4 class="benefits__title">Futuristic Design</h4>
+              <p class="benefits__text">To give Luna a truly flawless look, we specifically picked aircraft grade aluminum as its material and adopted both three-dimensional stretch-bending technology and a high precision cold forging technique.</p>
```

Comment:
move text to a new line, try to keep your code without horizontal scroll, fix in all places